radixdlt-engine/src/main/java/com/radixdlt/constraintmachine/REInstruction.java

```diff
@@ -218,7 +218,9 @@ Object read(REParser.ParserState parserState, ByteBuffer b) throws DeserializeEx
         return REFieldSerialization.deserializeSignature(b);
       }
     },
-    MSG((byte) 0xc, REOp.MSG, LengthType.VARIABLE, 1, 255) {
+    // There is no way to put actual limit here, so it is just lifted to maximum possible
+    // value, given used encoding format.
+    MSG((byte) 0xc, REOp.MSG, LengthType.VARIABLE, 1, 65535) {
```

Comment:
I think short is signed in Java, so we're restricted to half this

Reply:
This limit does not play a significant role, but you're right, there are no reason to set it higher than we can actually write.